pkg/server/module.go

```diff
@@ -1,19 +1,17 @@
 package server
 
 import (
+	"github.com/formancehq/go-libs/sharedotlp/pkg/sharedotlptraces"
 	"github.com/formancehq/webhooks/pkg/httpserver"
 	"github.com/formancehq/webhooks/pkg/storage/mongo"
-	"github.com/numary/go-libs/sharedotlp/pkg/sharedotlptraces"
 	"github.com/spf13/viper"
 	"go.uber.org/fx"
 )
 
 func StartModule(addr string) fx.Option {
 	var options []fx.Option
 
-	if mod := sharedotlptraces.CLITracesModule(viper.GetViper()); mod != nil {
-		options = append(options, mod)
-	}
+	options = append(options, sharedotlptraces.CLITracesModule(viper.GetViper()))
```

Comment:
Linter complains about condition always true. Which is the case with the updated version of the lib.